react/react1/week2/vite-project/src/App.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from 'react';
+import './App.css'
+import React from 'react';
+import TodoList from './components/TodoList/TodoList';
+
+const TODOS = [
+    {
+        id: 1,
+        description: "Get out of bed",
+    },
+    {
+        id: 2,
+        description: "Brush teeth",
+    },
+    {
+        id: 3,
+        description: "Eat breakfast",
+    },
+];
+
+
+function App() {
+    const [seconds, setSeconds] = useState(0);
+    const [isActive, setIsActive] = useState(true);
+    
+    /* 
+        Create a react timer using useEffect react hook
+    */
+    useEffect(() => {
```

Comment:
There is an easier way to do this. You want your seconds state to update once every second right, so if we make the use effect react to the seconds state, we'll have a useEffect running every second, so we can set a timeout of a second, and then increase seconds by one, making our useEffect run again :) Hope it makes sense

Reply:
> There is an easier way to do this. You want your seconds state to update once every second right, so if we make the use effect react to the seconds state, we'll have a useEffect running every second, so we can set a timeout of a second, and then increase seconds by one, making our useEffect run again :) Hope it makes sense

Hello Lucas:)
Thanks for your time and comments, I have gone through them but i did not understand this one.
Regards,
Jemima